cucumber/api/features/secrets_batch.feature

```diff
@@ -54,19 +54,27 @@ Feature: Batch retrieval of secrets
   Scenario: Fails with 404 if variable_ids param has some blank items
     When I GET "/secrets?variable_ids=cucumber:variable:secret1,,,cucumber:variable:secret2"
     Then the HTTP response status code is 404
+    And there is an error
+    And the error message includes "is empty or not found"
 
   Scenario: Fails with 404 if a variable_id param is of an incorrect format
     When I GET "/secrets?variable_ids=1,2,3"
     Then the HTTP response status code is 404
+    And there is an error
+    And the error message includes "is empty or not found"
 
   Scenario: Fails with 404 if a resource doesn't exist
     When I GET "/secrets?variable_ids=cucumber:variable:secret1,cucumber:variable:not-a-secret"
     Then the HTTP response status code is 404
+    And there is an error
+    And the error message includes "is empty or not found"
 
   Scenario: Fails with 404 if a resource doesn't have a value
     Given I create a new "variable" resource called "secret-no-value"
     When I GET "/secrets?variable_ids=cucumber:variable:secret1,cucumber:variable:secret-no-value"
     Then the HTTP response status code is 404
+    And there is an error
+    And the error message includes "is empty or not found"
```

Comment:
do you want to verify that the account is written as well? you can verify that the message includes `Variable cucumber:variable:secret-no-value is empty or not found`